src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -36,9 +36,8 @@ class HoldRequest extends React.Component {
   }
 
   render() {
-    const item = this.props.item;
     const searchKeywords = this.props.searchKeywords;
-    const record = this.props.item;
+    const record = this.props.bib;
     const title = record.title[0];
```

Comment:
That can be part of your PR with the tickets assigned to you to refactor! The point of this PR is just to resurface these pages because they were cut off a long time ago and we are now bringing them back.